library/include/hiptensor/internal/type_traits.hpp

```diff
@@ -270,6 +271,8 @@ namespace std
         hiptensor::detail::Fp16Bits eps(static_cast<uint16_t>(0x7FC0));
         return eps.b16;
     }
+#endif // DOXYGEN_SHOULD_SKIP_THIS
+    // @endcond
```

Comment:
Need the endcond?

Reply:
probably not. it snuck back in during a rebase

Reply:
removed it. could you please approve it again